src/Factories/TwigFactory.php

```diff
@@ -94,7 +95,9 @@ protected function getFunctions(): array {
 	 * @return AbstractExtension[]
 	 */
 	protected function getExtensions(): array {
-		return [];
+		return [
```

Comment:
Please move this to the MailerTemplatingFactory, we don't format the numbers in the web Twig.